vite.config.ts

```diff
@@ -41,7 +39,24 @@ export default defineConfig({
         }),
         cssInjectedByJsPlugin(),
     ],
+    resolve: {
+        alias: [
+            {find: '@', replacement: path.resolve(__dirname, '/src')},
+            {find: '#', replacement: path.resolve(__dirname, '/tests/unit')},
+            {find: '$', replacement: path.resolve(__dirname, '/tests/e2e')}
+        ],
+    },
+    test: {
+        globals: true,
+        environment: 'happy-dom',
+        include: [
+            'tests/unit/**/*.{test,spec}.{js,ts}'
+        ],
+        deps: {
+            inline: ['core/pkg'],
+        },
+    },
```

Comment:
This is again related to the environment. Vitest is a test framework to execute test for javascript code, based on vite.js. By default, Vitest does not run tests in a browser environment, but in a Node.js environment ([see doc](https://vitest.dev/guide/environment.html#test-environment)).

In the case of `aladin-lite`, you use browser specific APIs, such as `window` and more importantly `canvas`. Hence, we need to tell vitest to use a browser environment which will provide (fake) implementations of browser APIs.

There are two possibilities / libraries for doing so: `jsdom` and `happy-dom`. 

I couldn't remember why I chose `happy-dom` instead of `jsdom` apart from saying that for my website [arcsecond.io](https://www.arcsecond.io) (where I do a lot of things in a canvas), I finally ended up succeeding at making better tests with `happy-dom`. 

For `aladin-lite` we are just a the beginning of the testing codebase. I have no preference, and we will see later if `happy-dom` is good enough. We simply need one of the two.